internal/cluster/cluster_test.go

```diff
@@ -83,7 +96,84 @@ var _ = Describe("IsValidID", func() {
 			Expect(cluster.IsValidID("")).To(Not(Succeed()))
 		})
 	})
-})
+}
+
+func testSanitizeID() {
+	When("the id only contains alphabetic characters", func() {
+		It("should return same id", func() {
+			Expect(cluster.SanitizeID("abcdef")).To(Equal("abcdef"))
+		})
+	})
+
+	When("the id only contains numeric characters", func() {
+		It("should return same id", func() {
+			Expect(cluster.IsValidID("012345")).To(Succeed())
+			Expect(cluster.SanitizeID("012345")).To(Equal("012345"))
+		})
+	})
+
+	When("the id only contains alphanumeric characters", func() {
+		It("should return same id", func() {
+			Expect(cluster.SanitizeID("a0b1c2d3ef")).To(Equal("a0b1c2d3ef"))
```

Comment:
I wonder if it would make sense to extract this to a function e.g. 
```
func expectSanitizeIDNoChange(id string) {
	Expect(cluster.SanitizeID(id)).To(Equal(id))
}
```